Makefile

```diff
@@ -123,6 +154,96 @@ check.lint: ; $(info $(H) running golint on packages...)
 build: setup
 	bazel $(BAZEL_STARTUP_ARGS) build $(BAZEL_BUILD_ARGS) //...
 
+#-----------------------------------------------------------------------------
+# Target: go build
+#-----------------------------------------------------------------------------
+
+.PHONY: go-build
+
+.PHONY: pilot
+pilot: vendor
+	go install istio.io/istio/pilot/cmd/pilot-discovery
+
+.PHONY: pilot-agent
+pilot-agent: vendor
+	go install istio.io/istio/pilot/cmd/pilot-agent
+
+.PHONY: istioctl
+istioctl: vendor
+	go install istio.io/istio/pilot/cmd/istioctl
+
+.PHONY: sidecar-initializer
+sidecar-initializer: vendor
+	go install istio.io/istio/pilot/cmd/sidecar-initializer
+
+.PHONY: mixs
+mixs: vendor
+	go install istio.io/istio/mixer/cmd/mixs
+
+.PHONY: mixc
+mixc: vendor
+	go install istio.io/istio/mixer/cmd/mixs
+
+go-build: pilot istioctl pilot-agent sidecar-initializer mixs mixc
+
```

Comment:
It depends on `vendor` target, which calls `dep ensure -update` which does not work on a fresh install, because there's no `vendor` dir yet. So, for fresh installs we should call `dep ensure`. Therefore we could either modify the `vendor` target or instead depend on `depend` target.

Reply:
Got it - so far it seems it worked by accident :-), circleci is doing dep ensure before so vendor
exists. Will fix.